frontend/src/components/Buttons/LikeButton.vue

```diff
@@ -1,9 +1,14 @@
 <template>
   <v-btn
-    :icon="isFavorite ? IMdiHeart : IMdiHeartOutline"
```

Comment:
Why?

Reply:
The play pause button would not be centred in the middle without adding another button. I thought the like button made the most sense to add but to do that I needed to make it resizable otherwise it would be too small. I did that the same way that the other buttons that show on both the minimised media player and the full screen media player do it, but also added a default size to make it backwards compatible.

Reply:
Ah I see it now, I think I have figured it out, I wrote this code a while ago so my decision making process isn't super clear anymore but I think the reason why I am using v-if and v-else is because I used the shuffle button as a template which does it that way and I guess I didn't know any better, I'll fix it.